src/app/components/PasswordForm/index.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+  HiddenIcon,
+  VisibleIcon,
+} from "@bitcoin-design/bitcoin-icons-react/outline";
+import TextField from "@components/form/TextField";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+
+export type Props = {
+  keyPrefix: string;
```

Comment:
Should we name this `i18nKeyPrefix`? So people only looking at the props know what it's for?

Reply:
Done!